src/plugins/discover/public/application/main/utils/fetch_text_based.ts

```diff
@@ -41,6 +42,12 @@ export function fetchTextBased(
     time: timeRange,
     dataView,
     inputQuery,
+    titleForInspector: i18n.translate('discover.inspectorTextBasedRequestTitle', {
+      defaultMessage: 'ES|QL',
```

Comment:
```suggestion
      defaultMessage: 'Table',
```

Changing to "Table". Seems to fit.